bitcoin/src/bip152.rs

```diff
@@ -468,7 +468,7 @@ mod test {
             {
                 // test serialization
                 let raw: Vec<u8> = serialize(&BlockTransactionsRequest {
-                    block_hash: Hash::all_zeros(),
+                    block_hash: BlockHash::all_zeros(),
```

Comment:
In dd06be037e3dd11d1caba4836824ca830940803d:

As well as these (or they can join the `as_byte_array` commit) -- replacing `Hash` with concrete types.

Reply:
https://github.com/rust-bitcoin/rust-bitcoin/pull/2714